frontend/pages/outreach/index.js

```diff
@@ -0,0 +1,85 @@
+import Reach from 'react'
+
+import { LinkCards } from '../../components/LinkCards'
+
+const OutReachPage = () => {
+  return (
+    <>
+      {style}
+      <div className="container">
+        <h1>Outreach</h1>
+        <StreetHealth />
+        <StreetDoctor />
+      </div>
+    </>
+  )
+}
+
+const StreetHealth = () => {
+  return (
+    <>
+      <h2>Street Health</h2>
+      <blockquote>
+        <p>
+          Street Health is a service operated by Homeless Healthcare that
+          provides healthcare to rough sleepers in CBD parks in Perth and
+          Fremantle.
+        </p>
+      </blockquote>
```

Comment:
thinking about it, I am not quite sure if using a blockquote is more aesthetically pleasing, because I don't see a lot of articles that use blockquote to forward the paragraph.

Reply:
Eh, let the client can decide on that. It's ultimately subjective and the only ones we want to please are the clients.

Reply:
@JKKim98 definitely